components/prompt_based_laion_retrieval/requirements.txt

```diff
@@ -0,0 +1,2 @@
+fondant
```

Comment:
This uses the latest version by default, but the released version is frozen during the release process.

Reply:
nice